src/Logary/Internals.fs

```diff
@@ -205,6 +221,43 @@ module internal Set =
     | (s : _ Set) when s.Count = 0 -> Some EmptySet
     | _ -> None
 
+type Timeout =
```

Comment:
I wasn't aware that Hopac implements timeouts (other than the `timeOut` Alt, which I used for implementing `Job.withTimeout`). Could you point me in the right direction?